sdk/forms/src/main/java/com/klaviyo/forms/BridgeMessage.kt

```diff
@@ -0,0 +1,162 @@
+package com.klaviyo.forms
```

Comment:
Renamed this to `BridgeMessage` which is more in line with our overall naming conventions now, and I merged the `Utils` into the same file as a companion factory method.  

TODO meant to move the constants into the companion object too.